gh/github.go

```diff
@@ -0,0 +1,60 @@
+package gh
+
+import (
+	"PB173-discord-bot/gh/db"
+	"context"
+	"errors"
+	"github.com/bwmarrin/discordgo"
+	"github.com/google/go-github/v56/github"
+	"github.com/rs/zerolog/log"
+	"gorm.io/gorm"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	Client  *github.Client
+	Session *discordgo.Session
+	RepoDB  *gorm.DB
+)
```

Comment:
Create a struct out of this and pass it around.
```suggestion
type GitHubClient struct {
	Client  *github.Client
	Session *discordgo.Session
	RepoDB  *gorm.DB
}
```